app/bin/tools/delete_all_staging.dart

```diff
@@ -1,155 +0,0 @@
-// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
-// for details. All rights reserved. Use of this source code is governed by a
-// BSD-style license that can be found in the LICENSE file.
-
-import 'dart:async';
-
-import 'package:args/args.dart';
```

Comment:
@sigurdm do we have other tools for this?

Reply:
Not that I know - I also don't know if this tool is still functional (I have never tried it).

Reply:
Last when I restored to staging, I just imported on top - that is not really desirable, so we probably want a tool like this moving forward. However it could live in pub-ops.

Reply:
> However it could live in pub-ops.

I'll migrate it too with this PR.